catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventHandlerDelegate.java

```diff
@@ -348,7 +359,8 @@ public void error(SAXParseException exception) throws SAXException {
     @Override
     public void fatalError(SAXParseException exception) throws SAXException {
         String message = "Fatal Error: " + getParseExceptionInfo(exception);
```

Comment:
Actually...I'm going to leave that one. We pump it to the `outWriter` but also use it as the message on the exception.